### PR TITLE
Fabo/show all undelegations

### PR DESCRIPTION
**Pull request by faboweb**
Show undelegations as they are and not hide them when they are claimable
Show staked balance based on actual staking ledger and undelegations

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f61504c2533688f21c42a59--lunieio.netlify.app

File: api/lib/reducers/polkadotV0-reducers.js [hunk 1]
```diff
@@ -159,7 +160,7 @@ async function balanceV2Reducer(
     type: 'STAKE', // just a staking denom on Kusama for now
     available: availableLunieCoin.amount,
     total: totalLunieCoin.amount,
-    staked: stakedLunieCoin.amount,
+    staked: stakedLunieCoin.amount - undelegating,
```
Comment (Bitcoinera) [inline api/lib/reducers/polkadotV0-reducers.js:163]: oh woah, how could we miss that

File: app/src/components/staking/DelegationsOverview.vue [hunk 2]
```diff
@@ -90,18 +88,8 @@ export default {
...
-      // substract the already unbonding balance in the case of Substrate networks.
-      if (this.undelegationsLoaded && this.undelegations.length > 0) {
-        stakedAmount = this.undelegations.reduce(
-          (stakedAmount, { amount }) => stakedAmount - Number(amount),
-          stakedAmount
-        )
-      }
       return {
-        total: stakedAmount.toFixed(3),
+        total: stakingDenomBalance ? stakingDenomBalance.staked.toFixed(3) : 0,
```
Comment (Bitcoinera) [inline app/src/components/staking/DelegationsOverview.vue:92]: I'm a bit confused here. How does this fixes undelegations disappearing after reaching the claimable date?

File: api/lib/reducers/polkadotV0-reducers.js (hunk 1 above)
Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:163]: mmm, actually you just need to use active, no? It is the same

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:163]: For example:

```
"stash": "5GNtHmp1XmHJkGnM7swchrNGDcJ3r9yaQkumDZYK8tuEeZjG",
"total": 118362691275,
"active": 117362691275,
"unlocking": [
{
"value": 1000000000,
"era": 2717
}
],
```

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:163]: We could also include unstaking in balanceV2 actually :shrug: It is not a bad idea

File: app/src/components/staking/DelegationsOverview.vue (hunk 2 above)
Comment (faboweb) [reply app/src/components/staking/DelegationsOverview.vue]: you could read the PR description: Show staked balance based on actual staking ledger and undelegations

File: api/lib/reducers/polkadotV0-reducers.js (hunk 1 above)
Comment (faboweb) [reply api/lib/reducers/polkadotV0-reducers.js]: alright! let's use active. didn't see that. do we need to query for `unstaking` anywhere?

**Comment by faboweb** [comment]
maybe we should have a different endpoint `stake` for all staking queries

**Pull request merged**